lib/kino/input.ex

```diff
@@ -39,6 +39,13 @@ defmodule Kino.Input do
     %__MODULE__{attrs: attrs}
   end
 
+  @doc false
+  def duplicate(input) do
+    input.attrs
+    |> Map.drop([:ref, :id, :destination])
+    |> new()
```

Comment:
I am worried about this being fragile in the long term because `new()` may not handle all of its attrs. What about instead of calling `new()`, make both `duplicate` and `new` call an internal function that sets those fields?

Reply:
>  because new() may not handle all of its attrs

wdym? All other functions like `textarea` call new and it adds those three fields. Alternatively we could drop those fields directly in new and then do `duplicate(input), do: new(input.attrs)`?

Reply:
Ignore me. `new` is the function I was talking about. :)